### PR TITLE
Adds xdg-desktop-portal to the submodule list

### PR DESCRIPTION
**Pull request by luis-pereira**

### CODE REVIEW

**Comment by stefonarch** [comment]
Why is this not merged?

**Comment by tsujan** [comment]
> Why is this not merged?

Perhaps it isn't tested yet?

File: .gitmodules [hunk 1]
```diff
@@ -114,3 +114,7 @@
 [submodule "qtxdg-tools"]
 	path = qtxdg-tools
 	url = ../qtxdg-tools
+[submodule "xdg-desktop-portal-lxqt"]
+	path = xdg-desktop-portal-lxqt
+	url = ./xdg-desktop-portal-lxqt
```
Comment (yan12125) [inline .gitmodules:119]:
```suggestion
	url = ../xdg-desktop-portal-lxqt
```

**Review by yan12125 — CHANGES_REQUESTED** [review]
`CMakeLists.txt` should probably be updated as well

**Comment by yan12125** [comment]
Thanks for the update. I noticed a new branch https://github.com/lxqt/lxqt/tree/update-build, which seems related?

**Comment by luis-pereira** [comment]
> Thanks for the update. I noticed a new branch https://github.com/lxqt/lxqt/tree/update-build, which seems related?

Thanks for the review. Yes, it's related. It updates the build system. PR here: https://github.com/lxqt/lxqt/pull/2455

**Review by yan12125 — APPROVED** [review]

**Comment by yan12125** [comment]
Thanks! I will look into the other PR.

**Pull request merged**